app/Exceptions/CheckInCollisionException.php

```diff
@@ -19,4 +19,4 @@ public function __construct(TrainCheckin $trainCheckIn, $message = "", $code = 0
     public function getCollision(): TrainCheckin {
         return $this->trainCheckIn;
     }
-}
\ No newline at end of file
+}
```

Comment:
I'm confused... Codacy wants "1 newline at end of file; 0 found", but... there it is...?